src/style.css

```diff
@@ -0,0 +1,97 @@
+#x {
+  box-sizing: border-box;
+  font-family: Avenir, sans-serif;
+  width: 100%;
+  font-weight: 300;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  color: #3d4e61;
+  padding: 23px 62px;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+#x:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+#x:focus {
+  font-weight: 800;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+  outline: none;
+}
+
+@font-face {
+  font-family: Avenir;
+  font-style: normal;
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+}
+@font-face {
+  font-family: Avenir;
+  font-style: normal;
+  font-weight: 800;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+.icon {
+  position: absolute;
+  top: 25px;
+  left: 26px;
+}
+
+#y {
+  font-family: Avenir, sans-serif;
+  background-color: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  box-sizing: border-box;
+  display: block;
+  border-radius: 4px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 19px;
+  color: #3d4e61;
+  width: 100%;
+  padding: 11px 33px 10px;
+}
+
+.icon1 {
+  position: absolute;
+  top: 15px;
+  left: 13px;
+  width: 11px;
+}
+
+#y:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+#y:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  font-weight: 800;
+}
+
+#x::placeholder {
```

Comment:
Most of this styles are duplicated, there is no need to do it.
Create class for main input (for example big), then set only values that should be different for second class (for example for small input). Then just use two classes on small input

This will help to reduce your styles